server/project.go

```diff
@@ -263,7 +263,7 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 				URL:                    fmt.Sprintf("%s/%s/%s", s.profile.ExternalURL, gitlabWebhookPath, repositoryCreate.WebhookEndpointID),
 				SecretToken:            repositoryCreate.WebhookSecretToken,
 				PushEvents:             true,
-				PushEventsBranchFilter: repositoryCreate.BranchFilter,
+				PushEventsBranchFilter: "",    // For consistency with GitHub, we leave blank for all.
```

Comment:
What about just deleting this field from our codebase since we're not using it anymore?